service/src/main/java/io/spaship/operator/utility/EventAttribute.java

```diff
@@ -0,0 +1,12 @@
+package io.spaship.operator.utility;
+
+public interface EventAttribute {
+
+    //CRO ops and Preview event attributes
+    String CR_NAME = "cr_name:";
+    String NAMESPACE = "namespace:";
+    String MESSAGE = "message:";
```

Comment:
I would rather use enumeration `code` of actions rather than String representation of action as `message`.
By using `enum` with predefined codes it's consistent for both emitter and consumer to understand what kind of event it is.

Then for example Logging consumer would translate the code into the log message. The same for other consumers - they would translate the event code into appropriate action.

Reply:
Aggreed. As discussed made the changes.